src/vsr/replica.zig

```diff
@@ -1070,6 +1148,13 @@ pub fn ReplicaType(
             assert(message.header.view == self.view);
```

Comment:
```
            if (message.header.view > self.view) {
                log.debug("{}: on_commit: ignoring (newer view)", .{self.replica});
                return;
            }
```

Is this actually possible? We do a view_jump on commit, so our view should not be less?

Reply:
When handling `commit`/`prepare` messages, `view_jump()` does not update the `view` immediately, it just sends a `request_start_view`.

Reply:
Ah, yes, it is possible: view_jump won't actually jump on commit, it'll just request `start_view`.